swap_meet/item.py

```diff
@@ -1,2 +1,12 @@
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        descriptions = ["no comment", "eh", "okay", "good", "great"," perfect, mwah"]
+        return descriptions[int(self.condition)]
```

Comment:
Love this as a succinct solution that works for decimals as well as integers! I'd consider formatting the list a little differently to make it a bit easier to read, even through it take more lines:
```suggestion
    def condition_description(self):
        descriptions = [
            "no comment", 
            "eh", 
            "okay", 
            "good", 
            "great",
            "perfect, mwah"
        ]
        return descriptions[int(self.condition)]
```